js/fullWidthTreeView.js

```diff
@@ -60,12 +60,13 @@
     $('#fullwidth-treeview-row').resizable({handles: 's'}).animate({height: '200px'}, 500);
 
     // Scroll to active node
-    var active_node = null;
-    if ( active_node = $('li [selected_on_load]')[0])
-    {
-      active_node.scrollIntoView(true);
-      $('body')[0].scrollIntoView(true);
-    }
+    $('#fullwidth-treeview').bind('ready.jstree', function() {
+      var active_node = $('li[selected_on_load="true"]')[0];
+      if (active_node) {
```

Comment:
Hey, totally unrelated but can you take this chance to clean up that script to instantiate `$("#fullwidth-treeview")` only once?